CIP-XXXX/README.md

```diff
@@ -0,0 +1,312 @@
+Title: <Oracle Datum Standard>  
+Authors: Cardano DeFi Alliance <info@cardanodefialliance.org>, Cody Butz <cbutz@indigo-labs.io>, Jonas Lindgren <jonas@charli3.io>
+Comments-Summary: <No comments>  
+Comments-URI: <>  
+Status: <Draft>  
+Type: <Standards>  
+Created: <2022-10-20>  
+License: <CC-BY-4.0>
+---
+
+## Abstract
+
+This proposal defines a data standard for oracles to follow when placing data on-chain, as well as consumers to read the datum of produced oracle feeds.
+
+## Motivation
+
+Oracles provide a way for real-world data to be provided to a blockchain or smart contract to allow interaction with external data sources. Oracle's query, verify, and authenticate external data and then relay it to the blockchain. Oracles can provide multiple types of data feeds, for example: asset spot price, access to indexes, and statistics data for a particular blockchain.
+
+Oracles act as a data provider to the blockchain. As a data provider, it makes sense for the data feeds to be standardized so that they can be reused by multiple different projects, reducing fee overhead and decreasing feed setup overhead.
+
+Data consumers, who are reading the price data from an oracle, want to be able to support multiple data feeds from different providers for fall-back mechanisms and additional decentralization. If each oracle solution is providing a different datum/metadata structure, these particular data structures have to be supported by the data consumers' smart contracts and/or applications. By having a standard data structure for each particular data feed type, as a data consumer I can write my smart contracts/application to support that standard and expect the oracle to support that particular standard.
+
+Data consumers that share a common format for the data feeds they consume, may also be able to collaborate to provide more well-secured and frequently updated data feeds to share together, than they would otherwise be able to accomplish by each using their own standard.
+
+## Specification
+
+Oracles are responsible for providing multiple types of off-chain data to the blockchain. This proposal outlines an on-chain datum standard for Oracle producers to submit to the blockchain and Oracle consumers to read from the blockchain in a standardized format.
+
+Such a standard should accomplish a number of somewhat contradictory objectives
+
+* Data provided by the feed should be possible to alter over time, to add more detailed data or remove parts of the data, without major disruption to existing users of the data feed
+* Data provided in the feed should be stored in an efficient manner, to avoid data duplication and excessive transaction costs
+* The standard should be flexible enough to capture the majority of use-cases, even those that may not yet be conceived of at the writing of this document
+* The standard should provide room for individual data providers to add their own data fields to a feed, without interfering with the common standard
```

Comment:
We will always know more in the future than what we do now about how the standards are shaping up, but Charli3 (which I represent, in the interest of full disclosure) is already running free public feeds since several months back. It is in the Cardano community's interest that these feeds are provided in a broadly-useful and as standardized a way as possible, which is why we have had this discussion around standard for a long time prior to submitting this proposal, and are now requesting input from everyone else as to what is good and what needs improvement.

The sooner we begin this process, the sooner we can start to move towards something that is good. I don't imagine this standard will be perfect from the first draft and I think we should all be open to it evolving over time. Ideally in such a way that it remains backwards-compatible, that would be my biggest wish for the future.